commons/clever-tools/getting_started.md

```diff
@@ -16,24 +16,60 @@ applications from the command line with Clever Tools.
 
 ## Installing Clever Tools
 
-Clever Tools is packaged and distributed through [npm](https://npmjs.com).
-You need nodeJS, npm and openssl headers (openssl-dev on ubuntu / debian)
-installed on your system.
+Clever Tools is available on Windows, GNU/Linux and MacOS.
 
-To install Clever Tools for your user only, run
+### MacOS
 
-    npm install -g clever-tools
+Clever Tools is packaged using [homebrew](brew.sh):
 
-To install Clever Tools system-wide, run
+    brew install CleverCloud/tap/clever-tools
 
-    sudo npm install -g clever-tools
+If you don't want to use `brew`, a pre-compiled version is available: [clever-tools-0.9.0_macos.tar.gz](https://clever-tools.cellar.services.clever-cloud.com/releases/0.9.0/clever-tools-0.9.0_macos.tar.gz).
+You need to put both files (`clever` and `nodegit.node`) in your `PATH` to use the application.
 
-### Installing auto-completion
+#### Autocompletion
 
-Clever Tools comes with a powerful auto-completion support. Make sure to
-enable it for a better experience.
+The brew package enables autocompletion (for `bash` and `zsh`). Make sure `bash-completions` or `zsh-completions` are properly set up.
 
-    sudo install-clever-completion
+    # In ~/.bash_profile
+    . /usr/local/etc/bash_completion
+
+    # In ~/.zshrc
+    fpath=(/usr/local/share/zsh-completions $fpath)
+
+### Windows
+
+Clever Tools is packaged using [chocolatey](chocolatey.org):
+
+    choco install clever-tools
+
+If you don't want to use `chocolatey`, a pre-compiled version is available: [clever-tools-0.9.0_windows.zip](https://clever-tools.cellar.services.clever-cloud.com/releases/0.9.0/clever-tools-0.9.0_windows.tar.gz).
+You need to add both files (`clever.exe` and `nodegit.node`) to your `PATH` to use the application.
+
+### GNU/Linux
+
+A pre-compiled version is available: [clever-tools-0.9.0_linux.tar.gz](https://clever-tools.cellar.services.clever-cloud.com/releases/0.9.0/clever-tools-0.9.0_linux.tar.gz).
+You need to add both files (`clever` and `nodegit.node`) to your `PATH` to use the application.
+
+#### Autocompletion
+
+Clever Tools comes with a comprehensive auto-completion system.
```

Comment:
Not a big deal but why is this here and not in the Mac OS section as well?

Reply:
Autocompletion is installed by the brew package. I can mention it though